homeassistant/components/gios/strings.json

```
@@ -34,6 +34,56 @@
           "good": "Good",
           "very_good": "Very good"
         }
+      },
+      "no2_index": {
+        "state": {
+          "very_bad": "Very bad",
```

You can reference the ones from aqi. This removes the need for duplicate translations.

For example:
```suggestion
          "very_bad": "[%key:component::gios::entity::sensor::aqi::state:very_bad%]",
```